src/middleware/builtin.ts

```diff
@@ -284,12 +302,32 @@ function isCallbackIdentifiedAction(action: SlackAction<string>): action is Call
   return (action as CallbackIdentifiedAction).callback_id !== undefined;
 }
 
+type CallbackIdentifiedOption =
```

Comment:
super nitpicky of me, but i think this should be plural and so should the argument name in the following function, for consistency. if we want to go with singular, then lets do it everywhere.